public/components/custom_panels/home.tsx

```diff
@@ -56,14 +85,34 @@ export const Home = ({
   pplService,
   dslService,
   renderProps,
+  coreSavedObjects,
 }: PanelHomeProps) => {
   const [customPanelData, setcustomPanelData] = useState<CustomPanelListType[]>([]);
   const [toasts, setToasts] = useState<Toast[]>([]);
   const [loading, setLoading] = useState(false);
   const [toastRightSide, setToastRightSide] = useState<boolean>(true);
   const [start, setStart] = useState<ShortDate>('');
   const [end, setEnd] = useState<ShortDate>('');
-  const history = useHistory();
+
+  const savedObjectToCustomPanel = (
+    obj: SavedObject<ObservabilityPanelAttrs>
+  ): CustomPanelListType => {
+    return {
+      id: obj.id,
+      name: obj.attributes.title,
+      description: '',
+      dateCreated: obj.attributes.dateCreated,
+      dateModified: obj.attributes.dateModified || 0,
+      visualizations: obj.attributes.visualizations,
+      queryFilter: obj.attributes.queryFilter,
+      timeRange: obj.attributes.timeRange,
+      applicationId: obj.attributes.applicationId,
+    };
+  };
+
+  useEffect(() => {
+    console.log('CustomPanels:Home coreSavedObjects', { coreSavedObjects });
+  }, [coreSavedObjects]);
```

Comment:
If this is just fir debug logging let's remove this.

Reply:
Yes